Mathlib/AlgebraicGeometry/EllipticCurve/Group.lean

```diff
@@ -138,7 +138,7 @@ lemma YClass_ne_zero [Nontrivial R] (y : R[X]) : YClass W y ≠ 0 :=
 set_option linter.uppercaseLean3 false in
 #align weierstrass_curve.coordinate_ring.Y_class_ne_zero WeierstrassCurve.Affine.CoordinateRing.YClass_ne_zero
 
-lemma C_addPolynomial (x y L : R) : mk W (C (W.addPolynomial x y L)) =
+lemma C_addPolynomial (x y L : R) : mk W (C <| W.addPolynomial x y L) =
```

Comment:
Personally I prefer the previous spelling with the parenthesis, here I think it helps readability, but as you prefer.